docusaurus.config.js

```diff
@@ -55,6 +55,12 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      algolia: {
+        appId: 'Y6AWXACW2K',
+        apiKey: 'c7f1f2eb3086f5d2f2c2631f040263fb',
```

Comment:
Are these keys only used to query algolia for Epinio text (aka search)? Is it ok to be public like this?

Reply:
Could someone else used our keys to do search on other indices (see next line) and consume our free quota?

Reply:
yes, based on Algolia own documentation, these are the "public facing values" and are linked to our account that owns docs.epinio.io.
Therefore, there's no compromise of the account nor a possible way to "add/use" our quota.